crates/kubelet/src/store/mod.rs

```diff
@@ -72,3 +124,82 @@ pub trait Store {
             .collect()
     }
 }
+
+/// A `Store` implementation which obtains module data from remote registries
+/// but caches it in local storage.
+pub struct CachingStore<S: Storer, C: Client> {
```

Comment:
I thought about `Local` and rejected it because the 'local' store might not be actually local, e.g. a blob store.  But I guess the intent is about locality, that the private storage is nearby - and the doc comment already says local.  Yeah, all right.  Sold.